packages/lit-dev-content/site/docs/components/lifecycle.md

```diff
@@ -147,6 +147,22 @@ In more detail, it looks like this:
 
 <img class="centered-image" src="/images/docs/components/update-4.jpg">
 
+### The changedProperties map {#changed-properties}
+
+Many reactive update methods receive a `Map` of changed properties. The `Map` keys are the property names and its values are the previous property values. Changing a property *during* the update  (up to and including the `render()` method) updates the `changedProperties` map, but does not kick off a new update. Changing a property _after_ `render()` (for example, in the `updated()` method) triggers a new update cycle, and the changed property is added to a new `changedProperties` map to be used for he next cycle.
```

Comment:
Maybe move the mention of updating properties during an update to after the types section? It's kind of an edge case.

Reply:
Hmm... I was afraid if I put it after the types, folks who weren't using typescript would never get there. Maybe I'll make these two little H4 subsections--TypeScript types and Changing properties during an update

Reply:
Did that, I think it works pretty well.